scripts/zones/West_Ronfaure/npcs/Aaveleon.lua

```diff
@@ -4,22 +4,30 @@
 -- Involved in Quest: A Sentry's Peril
 -- !pos -431 -45 343 100
 -----------------------------------
+local ID = require("scripts/zones/West_Ronfaure/IDs")
 require("scripts/globals/npc_util")
 require("scripts/globals/quests")
 -----------------------------------
 
 function onTrade(player, npc, trade)
-    if player:getQuestStatus(SANDORIA, tpz.quest.id.sandoria.A_SENTRY_S_PERIL) == QUEST_ACCEPTED then
-        if npcUtil.tradeHas(trade, 600) then
-            player:startEvent(100)
-        else
-            player:startEvent(118)
-        end
+    if npcUtil.tradeHas(trade, 600) then -- Ointment
+        player:startEvent(100) -- He accepts the ointment and gives the player the empty case to return to his wife.
+    elseif player:getQuestStatus(SANDORIA, tpz.quest.id.sandoria.A_SENTRY_S_PERIL) < QUEST_COMPLETED then
+        player:startEvent(106) -- "What's this? I can't accept gifts from strangers."  He stops saying this after quest complete.
```

Comment:
does he just not respond to trades post quest completion?

Reply:
Yes, Nyu confirmed.